autogen/img_utils.py

```diff
@@ -298,3 +310,51 @@ def message_formatter_pil_to_b64(messages: List[Dict]) -> List[Dict]:
         new_messages.append(message)
 
     return new_messages
+
+
```

Comment:
Should the LLM config have a field that explicitly states that the model is multimodal? I can see name detection going wrong very easily

Reply:
Yes, we have a parameter "vision-model" in the LLM config, which can be used to declare it is a vision model.

Let me create an example in the notebook.